paper2remarkable/ui.py

```diff
@@ -19,16 +19,21 @@
 from . import __version__
 from .exceptions import InvalidURLError
 from .exceptions import UnidentifiedSourceError
-from .providers import LocalFile
+from .providers import LocalFile, EPUBProvider
 from .providers import providers
 from .utils import follow_redirects
 from .utils import is_url
 
 
 def build_argument_parser():
     parser = argparse.ArgumentParser(
-        description="Paper2reMarkable version %s" % __version__
+        description="Paper2reMarkable version %s - Upload PDFs and EPUBs to reMarkable" % __version__
```

Comment:
Nit: Please don't change the description, the format is modeled on command command line tools (e.g., `git -v` )